test/CameraCaptureUITests/wil.natvis

```diff
@@ -0,0 +1,170 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+    Copyright (c) Microsoft. All rights reserved.
+    This code is licensed under the MIT License.
+    THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF
+    ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED
+    TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+    PARTICULAR PURPOSE AND NONINFRINGEMENT.
+-->
+<AutoVisualizer xmlns="http://schemas.microsoft.com/vstudio/debugger/natvis/2010">
```

Comment:
I'm not sure the WIL natvis file belongs in this project...

If you want it embedded in your PDB you should be able to pull it out of the WIL nuget package directory.

Reply:
I understand your point. However, I've noticed that every test project in this repo includes a wil.natvis file, and I figured it would be consistent to add it as well.